components/TheLabTier.vue

```diff
@@ -0,0 +1,65 @@
+<script setup>
+import VFlexibleLink from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VFlexibleLink.vue'
+const props = defineProps({
+  icon: {
+    type: String,
+    default: '',
+  },
+  name: {
+    type: String,
+    default: '',
+  },
+  cost: {
+    type: String,
+    default: '',
+  },
+  joinLink: {
+    type: String,
+    default: "https://members.radiolab.org/",
+  },
+})
+
+//const emit = defineEmit(["change", "click"]);
```

Comment:
please remove commented out code. It's fine to keep it on your working branch, but this shouldn't make it into `main`